css/style.css

```diff
@@ -0,0 +1,276 @@
+/*  Add your styling here */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+/* ----- Header ----- */
+
+.header {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  background-color: #eee;
+  padding: 20px;
+  border-bottom: 1px solid #eee;
+}
+.logo-img {
+  grid-row: 1 / -1;
+  margin-left: 10px;
+}
+.logo-img img {
+  display: grid;
+  width: 100px;
+  opacity: 50%;
+}
+
+.header-content {
+  grid-column: 2 / -1;
+  font-size: 20px;
+  font-weight: 300;
+  font-style: italic;
+  color: #979797;
+  direction: rtl;
+  margin-top: -20px;
+}
+
+/* ----- Hamburger Menu ----- */
+.hamburger-container {
+  justify-self: end;
+}
+.hamburger {
+  display: block;
+  padding: 5px 5px;
+  margin: -10px 0 auto 0;
+  cursor: pointer;
+}
+.bar {
+  display: block;
+  width: 25px;
+  height: 4px;
+  margin: 5px auto;
+  background-color: #979797;
+}
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+/* ----- Nav Menu ----- */
+.nav-menu {
+  position: fixed;
```

Comment:
This works, but when you scroll the page on a small screen, the open menu covers the content, due to `position: fixed`.
Alternative method: set `position: relative` on the `.header` container, and use `position:absolute` on the popup menu.